main.js

```diff
@@ -0,0 +1,17 @@
+/* exported data */
+
+var $car = document.querySelector('img');
+
+window.addEventListener('keydown', turnCar);
+
+function turnCar(event) {
+  if (event.keyCode === 40) {
+    $car.style.transform = 'rotate(90deg)';
+  } else if (event.keyCode === 37) {
+    $car.style.transform = 'rotate(180deg)';
+  } else if (event.keyCode === 38) {
+    $car.style.transform = 'rotate(-90deg)';
+  } else if (event.keyCode === 39) {
+    $car.style.transform = 'rotate(0deg)';
```

Comment:
You should make a class for each of these `transform` styling adjustments, and just change the class of the element to the class for that direction.